package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "codeceptjs-browserstack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "single": "npx codeceptjs run -c src/test/single/codecept.conf.js --steps",
+    "parallel": "npx codeceptjs run-multiple bstack -c src/test/parallel/codecept.conf.js --steps",
+    "local": "npx codeceptjs run -c src/test/local/codecept.conf.js --steps"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/princebaretto99/codeceptJS-browserstack.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/princebaretto99/codeceptJS-browserstack/issues"
+  },
+  "homepage": "https://github.com/princebaretto99/codeceptJS-browserstack#readme",
+  "devDependencies": {
+    "codeceptjs": "^3.2.3",
+    "webdriverio": "^6.12.1"
```

Comment:
Any reason for this particular version of webdriverIO (asking since this is neither the latest nor the most used version)?